subworkflows/local/vcf_normalization/main.nf

```diff
@@ -0,0 +1,49 @@
+// Normalize all unannotated VCFs
+
+// Import modules
+include { ADD_INFO_TO_VCF  } from '../../../modules/local/add_info_to_vcf/main'
+include { TABIX_BGZIPTABIX as TABIX_VCF } from '../../../modules/nf-core/tabix/bgziptabix/main'
+include { BCFTOOLS_NORM as VCFS_NORM } from '../../../modules/nf-core/bcftools/norm/main'
+include { BCFTOOLS_SORT as VCFS_SORT } from '../../../modules/nf-core/bcftools/sort/main'
+include { TABIX_TABIX as TABIX_VCFS_INDEX } from '../../../modules/nf-core/tabix/tabix/main'
+
+// Workflow to normalize, compress, and index VCF files
+workflow NORMALIZE_VCFS {
+
+    take:
+    vcfs
+    fasta
+
+    main:
+    versions = Channel.empty()
+
+    // Add additional information to VCF files
+    ADD_INFO_TO_VCF(vcfs)
```

Comment:
We are doing the same thing in the concatenation step. Can you check what happens if someone concatenates and the normalisaes? I have the feeling this will end a bunch of redundant information. On that note, the current order is: concat then normalise. Are we sure it shouldn't be the other way around?

Reply:
In the original PR, there was a vcf_concatenate process that performed normalization first and then concatenation of germline vcfs. Since I wanted to normalize all vcfs without concatenation, I decided to keep the original process and add an additional one (vcf_normalization/main.nf) specifically for normalization. Also, Sarek already includes a process for concatenating germline vcfs, so I thought it should remain unchanged.

However, I am a bit confused because you mentioned that concatenation occurs before normalization. I initially thought that based on the boolean parameters (params.concatenate, params.normalized_vcfs), one could choose which process to run. If both processes run, I expected two different outputs: concatenated and normalized germline vcfs, and normalized vcfs (for all). It seems like I might have misunderstood the intended workflow.

Could you please explain where the order of processes comes from?